File: JavaServer/src/main/java/com/vikreta/dto/LoginResponse.java
package com.vikreta.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponse {
	private Long id;
	private String firstName;
	private String jwt;
	private String message;
	private String role;
	private boolean success;
}

File: .NetServer/Vikreta/DTO/LoginResponse.cs
using System.Runtime.InteropServices;
using Vikreta.Entities;

namespace Vikreta.DTO
{
    public class LoginResponse
    {
        
       

        public long id { get; set; }

        public string firstName { get; set; }

        public string jwt { get; set; }

        public string role { get; set; }

        public bool success { get; set; }

        public LoginResponse (long id , string firstName, string jwt , string role, bool success)
        {

            this.id = id;
            this.firstName = firstName;
            this.jwt = jwt;
            this.role = role;
            this.success = success;

        }
    }

}
